admin/widgets.rb

```diff
@@ -0,0 +1,171 @@
+      f.input :active, label: I18n.t('active_admin.widget.label3'), hint: I18n.t('active_admin.widget.hint3')
+      f.input :default, label: I18n.t('active_admin.widget.label4'), hint: I18n.t('active_admin.widget.hint4')
+    end
+    f.inputs I18n.t('active_admin.widget.layout_web'), class: "foldable inputs" do
+      f.input :content, label: I18n.t('active_admin.widget.label_web'), hint: I18n.t('active_admin.widget.hint_web')
+    end
+    # Wird so gut wie nie benutzt => erstmal ausgeblendet:
+    # f.inputs I18n.t('active_admin.widget.layout_mobile'), class: "foldable inputs closed" do
+    #   f.input :mobile_content, label: I18n.t('active_admin.widget.label_mobile'), hint: I18n.t('active_admin.widget.hint_mobile')
+    # end
+    f.inputs I18n.t('active_admin.widget.info'), class: "foldable inputs closed"  do
+      f.input :sorter, label: I18n.t('active_admin.widget.label_info1'), hint: I18n.t('active_admin.widget.label_hint1')
+      f.input :css_name, label: I18n.t('active_admin.widget.label_info2'), hint: I18n.t('active_admin.widget.label_hint2')
+      f.input :id_name, label: I18n.t('active_admin.widget.label_info3'), hint: I18n.t('active_admin.widget.label_hint3')
+      # Wird so gut wie nie benutzt => erstmal ausgeblendet:
+      #f.input :teaser
+      #f.input :description, label: I18n.t('active_admin.widget.label_info4'), hint: I18n.t('active_admin.widget.label_hint4')
+    end
+    f.inputs I18n.t('active_admin.widget.access_rights'), class: "foldable closed inputs" do
+      f.has_many :permissions do |p|
+        p.input :role, include_blank: "Alle"
+        p.input :action, as: :select, collection: Goldencobra::Permission::PossibleActions, include_blank: false
+        p.input :_destroy, as: :boolean
+      end
+    end
+    f.inputs I18n.t('active_admin.widget.article') do
+      f.input :articles, label: I18n.t('active_admin.widget.article_label'), hint: I18n.t('active_admin.widget.article_hint'), as: :select, collection: Goldencobra::Article.order("title ASC"), input_html: { class: 'chosen-select', "data-placeholder" => I18n.t('active_admin.widget.article_placeholder') }
+    end
+    f.actions
+  end
+
+  sidebar :layout_positions, only: [:edit] do
+    ul do
+      Goldencobra::Widget.tag_counts_on(:tags).map(&:name).each do |wtag|
+        li do
+          wtag
+        end
+      end
+    end
+  end
+
+  sidebar :help, only: [:edit, :show] do
+    render "/goldencobra/admin/shared/help"
+  end
+
+  index download_links: proc{ Goldencobra::Setting.for_key("goldencobra.backend.index.download_links") == "true" }.call do
+    selectable_column
+    column I18n.t('active_admin.widget.title'), :title, sortable: :title do |widget|
+      link_to(widget.title, edit_admin_widget_path(widget), title: I18n.t('active_admin.widget.title1'))
+    end
+    column I18n.t('active_admin.widget.position'), :tag_list, sortable: false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.